src/visitors/dto/visitorDto.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { IsAlpha, IsEnum, IsNotEmpty, IsPhoneNumber, IsString } from "class-validator";
+import { Gender } from "src/crm/enums/gender";
+import { WHLocation } from "../enums/whLocation";
+
+//DTO to return the visitor information
```

Comment:
I think this comment should: to validate visitor information, could you please edit it